Makefile

```diff
@@ -33,14 +44,43 @@ logs: ## display app logs (follow mode)
 	@$(COMPOSE) logs -f app
 .PHONY: logs
 
-run: ## start the development server
+run: ## start the wsgi (production) or development server
 	@$(COMPOSE) up -d
 .PHONY: run
 
 stop: ## stop the development server
 	@$(COMPOSE) stop
 .PHONY: stop
 
+# == Frontend
+build-front: install-front build-sass ## build front-end application
+.PHONY: build-front
+
+build-sass: ## build Sass files to CSS
+	@$(YARN) sass
+.PHONY: build-sass
+
+install-front: ## install front-end dependencies
+	@$(YARN) install;
+.PHONY: install-front
+
+lint-front: lint-front-prettier ## run all front-end linters
```

Comment:
A preriquisite is missing or this task is useless. Just executing `lint-front` do nothing more than executing `lint-front-prettie`

Reply:
I'm planning for the future dude.

Reply:
YAGNI

Reply:
Consider this as an alias.